dynamic-config/config-cc/src/main/java/org/apache/servicecomb/config/client/ParseConfigUtils.java

```diff
@@ -74,14 +74,23 @@ public void initWithUpdateHandler(UpdateHandler updateHandler) {
   public void refreshConfigItems(Map<String, Map<String, Object>> remoteItems) {
     try {
       configLock.lock();
+      String oldRevision = currentVersionInfo;
+      boolean newVersion = false;
       currentVersionInfo =
           remoteItems.getOrDefault("revision", new HashMap<>()).getOrDefault("version", "default").toString();
       //make sure the currentVersionInfo != ""
       currentVersionInfo = currentVersionInfo.equals("") ? "default" : currentVersionInfo;
-      remoteItems.remove("revision");//the key revision is not the config setting
+      if (remoteItems.containsKey("revision")) {
```

Comment:
impossible. 
in the new version, when oldRevision is same with newRevision, it will returen 304
in the old version, it will not contain key revision

Reply:
so when got 304, also need to invoke refresh?

Reply:
of course not.
it will not invoke this method at all.

Reply:
why?
from the code, if newVersion is false, still invoked doRefreshItems

Reply:
ok, it's for old CC